src/database/config/config.js

```diff
@@ -0,0 +1,13 @@
+require('dotenv').config();
```

Comment:
You have to use ES6 syntax, you can wait for someone who will implement the express server he has also to configure babel so you that you can use the ES6 syntax.